pipeline/tasks/gapic_tasks.py

```diff
@@ -61,10 +61,7 @@ def _move_to(self, gapic_config_dir, gapic_api_yaml):
         elif len(gapic_api_yaml) == 0:
             raise ValueError(error_fmt + 'No location specified')
         conf_out = os.path.abspath(gapic_api_yaml[0])
-        if os.path.exists(conf_out):
-            raise ValueError(error_fmt + 'File already exists')
```

Comment:
I was just doing this the other day when using config gen to auto-generate the new IAM methods and copy them onto the existing gapic yaml. Admittedly, I had to hack the gapic yaml path so that it would generate to a different file path than the one that is checked in, so that it wouldn't get overwritten.